lib/bsdlib/Kconfig

```diff
@@ -5,13 +5,17 @@
 
 menu "BSD Library for nrf91"
 
+config BSD_LIBRARY_ALLOW_SECURE
+	bool "Allow usage of BSD library in secure(Experimental)"
+	default n
```

Comment:
IMO we can skip this as it is the default

Reply:
Skip the whole option? We need something to notify BSDLib that you are now going to allow running it without TrustZone(the user knows what he's doing) option.

If not we could just remove the TrustZone requirement but I'm afraid we are creating a foot gun by allowing users who don't understand the implications of doing this that their RAM will be split etc.

Reply:
No not the option, just the `default n` part